### PR TITLE
NEPT-497: Removed unused field_news_picture field - fix(Maintenance)

### PR DESCRIPTION
**Pull request by imanoleguskiza**

### CODE REVIEW

**Comment by gillesdeudon** [comment]
In the `news_og` sub-module, there is a views_alter site where the field is re-injected in the views for sites using O.G. (see profiles/common/modules/features/news/news_og/news_og.views_alter.inc, line 93).

Should it be removed as well?

**Comment by gillesdeudon** [comment]
:+1:

**Comment by drupol** [comment]
The fix itself is ok.

I do not agree on getting out of scope by adding: `$handler->display->display_options['path'] = 'news';`

This addition is also done in an alter function. If this path is meant to be fixed, why defining it in an alter function. This is something that I don't understand yet. I'm investigating.

File: profiles/common/modules/features/news/news_core/news_core.views_default.inc [hunk 1]
```diff
@@ -103,6 +103,7 @@ function news_core_views_default_views() {
   $handler->display->display_options['row_plugin'] = 'fields';
   $handler->display->display_options['defaults']['row_options'] = FALSE;
   $handler->display->display_options['defaults']['header'] = FALSE;
+  $handler->display->display_options['path'] = 'news';
```
Comment (drupol) [inline profiles/common/modules/features/news/news_core/news_core.views_default.inc]: That's my boy !

**Comment by drupol** [comment]
This makes more sense now. Giving my thumb up.

**Comment by drupol** [comment]
👍

**Comment by gillesdeudon** [comment]
:+1:

**Comment by drupol** [comment]
👍

**Comment by gillesdeudon** [comment]
:+1:

**Comment by gillesdeudon** [comment]
:+1:

**Comment by drupol** [comment]
:+1:

**Pull request merged**